ci/install_scripts/install_retinalOCT_RPD_segmentation_dependency.sh

```diff
@@ -0,0 +1,3 @@
+# ensure that using evironment with python==3.9
+pip install -r ./docs/requirements.txt #includes monai
```

Comment:
We may not need the requirements.txt file in the bundle, it's content is included in the `metadata.json`.
Our CI can generate the requirements file directly via: https://github.com/Project-MONAI/model-zoo/blob/dev/ci/get_bundle_requirements.py#L54